sdk/search/search-documents/src/serviceUtils.ts

```diff
@@ -42,6 +42,10 @@ import {
 } from "./serviceModels";
 
 export function convertSkillsToPublic(skills: SkillUnion[]): Skill[] {
+  if (!skills) {
```

Comment:
technically the function should be

```
export function convertSkillsToPublic(skills?: SkillUnion[]): Skill[] | undefined {
```

if we want to support calling it with undefined. Same as below. I'm not sure if TS 3.9 will complain here or not.

Reply:
An alternative would be to do the check in the caller

Reply:
Regarding the listSkillsetNames, I am still finalizing it with Heath. It should be finalized today. 

TS will complain in that case because the definition of Skills in SkillSet  is not optional. That is the reason, I settled for this :-)